guides/contexts.md

```diff
@@ -789,56 +793,58 @@ We defined a new `CartItemController` with the create and delete actions that we
 Let's implement our new interface of `ShoppingCart` context API in `lib/hello/shopping_cart.ex`:
 
 ```elixir
-  alias Hello.Catalog
-  alias Hello.ShoppingCart.{Cart, CartItem}
-
-  def get_cart_by_user_uuid(user_uuid) do
-    Repo.one(
-      from(c in Cart,
-        where: c.user_uuid == ^user_uuid,
-        left_join: i in assoc(c, :items),
-        left_join: p in assoc(i, :product),
-        order_by: [asc: i.inserted_at],
-        preload: [items: {i, product: p}]
-      )
-    )
-  end
++ alias Hello.Catalog
+- alias Hello.ShoppingCart.Cart
```

Comment:
duplicate alias

Reply:
oh, that's actually a delete line

Reply:
ah, it's originally showing diff. I beg your pardon.